src/golang/cmd/server/server/aqueduct_server.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const (
-	RequiredSchemaVersion = 8
```

Comment:
wow where is this field used? I'm surprised that nothing has been broken so far

Reply:
Well this was more important in the enterprise project. Here once we modify the `install_local` and packaging scripts, we know that the next release will be running the correct schema version.

We do have a check here https://github.com/aqueducthq/aqueduct/blob/main/src/golang/cmd/server/server/aqueduct_server.go#L118 just to confirm.